### ISSUE TITLE
Audible of Visual warning if batttery is about to give out. 

### ISSUE THREAD

**Issue by develroo** [clockworkpi/launcher#106]
Hey guys, 

Having great fun playing with the gameshell however just got a little miffed when I just completed a boss fight and was about to save when the battery gave out. Arrrg

So is there a chance that there can be some sort of audible warning or a quick flash on screen when the battery is say 10% or 5% and needs charging sooner not later?

Would certainly cut down on frustration , trust me. :)

Thanks for such a compelling product.

**Comment by cuu** [comment]
yeah ,you are right

In fact, a small widget called gsnotify can do this function very early, but the on-top window conflicts with twm(the wm of gameshell now) , causing focus-stealing problems 


I will  write a tutorial on how to set up a new version of launcher and enable this feature very soon
saw your post on forum

:)

**Comment by develroo** [comment]
Yeah I commented [on the forums](https://forum.clockworkpi.com/t/audible-or-visual-warning-if-the-batttery-is-about-to-give-out/1757) and someone mentioned this [battery monitor](https://forum.clockworkpi.com/t/app-battery-status-on-emulators/1272) and that works pretty well. 

It would be nice for something like this by default though.
